compiler/rustc_parse_format/src/lib.rs

```diff
@@ -9,6 +9,9 @@
     html_playground_url = "https://play.rust-lang.org/",
     test(attr(deny(warnings)))
 )]
+#![cfg_attr(not(bootstrap), doc(rust_logo))]
+#![cfg_attr(not(bootstrap), allow(internal_features))]
+#![cfg_attr(not(bootstrap), feature(rustdoc_internals))]
```

Comment:
```
// We want to be able to build this crate with a stable compiler, so no
// `#![feature]` attributes should be added.
```
This broke the crate from compiling on stable

Reply:
For crates like these it'd be nice if there was a `#[forbid(feature_attribute)]` lint or something

Reply:
I don't think this happened yet? Can I just delete the three lines?